### DISCUSSION TITLE
Restart required after Log-Level change? Log level deletion?

### DISCUSSION THREAD

**Discussion by Muffexx** [Chia-Network/chia-blockchain#3306] [Q&A] [answered]
Hello there, 

if have two questions.

1. Do I need to restart chia after changing the log-level in the yaml-file?
2. Will older logs be deleted automatically? 

I am curious because I wan't to see wether I'm replying fast enough. I heard that INFO logs quite much and the space on my main drive is quite limited.

regards
Muffexx

**Comment by ilylx** [comment, marked as answer]
1. Yes
2. Yes

**Reply by Muffexx** [reply]
Thanks!
Do you know where the log is deleted based on time or file-size? 
If have about 4GB of spare space left.

**Reply by kswtch** [reply]
log files are rotated by chia and eventually deleted. check config.yaml for options.

**Reply by Muffexx** [reply]
Thanks.
I couldn't find it in the yaml, but I found this in the Changelog:
"Log rotation should now properly rotate every 20MB and keep 7 historical logs."
